.github/workflows/docker-ort-runtime-ext.yml

```diff
@@ -169,7 +169,7 @@ jobs:
 
       - name: Get ORT current version
         run: |
-          ORT_VERSION=$(./gradlew -q properties --property version | sed -nr "s/version: (.+)/\1/p")
+          ORT_VERSION=$(./gradlew -q printSemVersion)
```

Comment:
Note that when being exactly on tag `2.0.0` (with a clean working tree), this would print `2.0.0+sha.5893bd7` instead of `2.0.0`. Is this desired?

Reply:
I wasn't aware of that, but I think it's fine.

Reply:
That would be a mismatch, however, for people running ORT also from the distribution archive, which shows `2.0.0` in this case. I'll look into [customizing the version format](https://github.com/jmongard/Git.SemVersioning.Gradle?tab=readme-ov-file#version-format) to address that.

Reply:
Shall we merge now this and then customize the version format in that gradle tasks later on?

Reply:
My alternative proposal is already [here](https://github.com/oss-review-toolkit/ort/pull/7625=. That could be used instead of this PR (not in addition to it). The advantage is that also non-Docker user would get the SHA1 info for pre-release builds.

Reply:
I can minimize it direct in the tagging, i can capture the sha without need to alter the current Gradle process.
But will solve only for the images in upstream, not the local build script/.